.travis.yml

```diff
@@ -24,6 +24,14 @@ jobs:
   - name: Reactive-sandbox test
     script:
     - make test CHARTS=reactive-sandbox
+  - stage: deploy
+    name: Deploy to GCS
+    # Only deploy on a push to master
+    if: type = push
```

Comment:
I'm not sure this is right. See https://github.com/lightbend/es-console-spa/blob/master/.travis.yml#L24 for what I used in es-console-spa.

Reply:
Hmm I guess this is the same thing as `(NOT type IN (pull_request))` ?

Reply:
I thought that test was appropriate, given 
https://docs.travis-ci.com/user/conditions-v1#integration
We only want to publish on a `push` right?  (I don't really know what`api` or `cron` Travis events are though.)

Reply:
api is if someone triggers the build manually. Cron is if we enable cron runs. We probably want it building on manual trigger, right? In case we need to rerun it.